src/main.cpp

```diff
@@ -537,7 +537,7 @@ class $modify(CCLayerColor)
 			this->getColor() == color2)
 		{
 			this->setColor({ 80, 80, 80 });
-			if (this->getChildrenCount() > 0) {
+			if (this->getChildrenCount() > 1) {
 				if (auto text = typeinfo_cast<CCScale9Sprite*>(this->getChildren()->objectAtIndex(1)) text->setColor({ 44, 44, 44 });
```

Comment:
Idk why but when i try to merge this it have syntax errors, but np, i fix the sintax on my commit, tysm

Reply:
Because I fucked up the syntax